lib/administrate/field/time.rb

```diff
@@ -0,0 +1,11 @@
+require_relative "base"
+
+module Administrate
+  module Field
+    class Time < Base
+      def default
+        data && data.strftime("%H:%M:%S")
```

Comment:
What about adding this format as described in http://guides.rubyonrails.org/i18n.html#adding-date-time-formats?